tests/unit/initializers/ember-feature-flags-test.js

```diff
@@ -32,7 +32,7 @@ test('service has application injected', function(assert) {
   assert.ok(service.application, 'service has application');
 });
 
-['route', 'controller', 'component'].forEach(function(type) {
+['controller', 'component'].forEach(function(type) {
```

Comment:
Tests for routes are failing starting Ember.js 2.10 with `Attempting to inject an unknown injection: 'service:Document'` error.

See https://travis-ci.org/SergeAstapov/ember-feature-flags/builds/330273229 for more details